lib/dynamoid/document.rb

```diff
@@ -129,6 +141,25 @@ def update(hash_key, range_key_value = nil, attrs)
         model
       end
 
+      # Update document.
+      # Uses efficient low-level `UpdateItem` API call.
+      # Changes attibutes and loads new document version with one API call.
+      # Doesn't run validations and callbacks. Can make conditional update.
+      # If a document doesn't exist or specified conditions failed - returns `nil`
+      #
+      # @param [Scalar value] partition key
+      # @param [Scalar value] sort key (optional)
+      # @param [Hash] attributes
+      # @param [Hash] conditions
+      #
+      # @return [Dynamoid::Document/nil] updated document
+      #
+      # @example Update document
+      #   Post.update(101, read: true)
+      #
+      # @example Update document with condition
+      #   Post.update(101, { read: true }, if: { version: 1 })
+
```

Comment:
Feel like it's nicer when documentation doesn't have blank line between it and the method.